src/HelpText.cpp

```diff
@@ -27,6 +27,14 @@
 #include "AllThemeResources.h"
 #include "Theme.h"
 
+#ifdef HAS_WHATS_NEW
+
+namespace
+{
+const char* WhatsNewURL = "https://youtube.com";
```

Comment:
Do we want to link to the youtube video directly, or link to something on audacityteam.org which 301-redirects to the appropriate video? Doing the latter would allow us to replace the video in case of errors in the video, or us/YouTube deleting the video accidentally.

Reply:
Well, we will change the link for sure. I wasn't aware that WP allows 301 redirects easily (and I'm still not sure, if it is possible)